package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "azuredatastudio",
   "version": "1.24.0",
-  "distro": "3b5783b2180557bf55e44b2b873cd47d4bd6ccd0",
+  "distro": "53b18168b6eda6ba96e69e504d6fd56a66ae15e0",
```

Comment:
We have the insiders icons in the repo as well so you should probably update those too